bigtable/examples/run_examples_utils.sh

```diff
@@ -0,0 +1,195 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+readonly CBT_CMD="${CBT:-${GOPATH}/bin/cbt}"
+readonly CBT_EMULATOR_CMD="${CBT_EMULATOR:-${GOPATH}/bin/emulator}"
+
+# Run all the instance admin examples against production.
+#
+# This function allows us to keep a single place where all the examples are
+# listed. We want to run these examples in the continuous integration builds
+# because they rot otherwise.
+function run_all_instance_admin_examples() {
+  if [ ! -x ../examples/bigtable_samples_instance_admin ]; then
+    echo "Will not run the examples as the examples were not built"
+    return
+  fi
+
+  local project_id=$1
+  shift
+  local zone_id=$1
+  shift
+
+  # Create a (very likely unique) instance name.
+  local -r INSTANCE="in-$(date +%s)"
+
+  local ignore_unimplemented="/bin/false"
+  if [ ! -z "${BIGTABLE_EMULATOR_HOST:-}" ]; then
+    ignore_unimplemented="echo Ignoring error as the emulator does not implement this function"
+  fi
+
+  echo
+  echo "Run create-instance example."
+  ../examples/bigtable_samples_instance_admin create-instance "${project_id}" "${INSTANCE}" "${zone_id}" || ${ignore_unimplemented}
+
+  echo
+  echo "Run list-instances example."
+  ../examples/bigtable_samples_instance_admin list-instances "${project_id}" || ${ignore_unimplemented}
+
+  echo
+  echo "Run get-instance example."
+  ../examples/bigtable_samples_instance_admin get-instance "${project_id}" "${INSTANCE}" || ${ignore_unimplemented}
+
+#  echo
+#  echo "Run list-clusters example."
+#  ../examples/bigtable_samples_instance_admin list-clusters "${project_id}" || ${ignore_unimplemented}
+
+  echo
+  echo "Run delete-instance example."
+  ../examples/bigtable_samples_instance_admin delete-instance "${project_id}" "${INSTANCE}" || ${ignore_unimplemented}
+}
+
+# Run all the instance admin examples against production.
+#
+# This function allows us to keep a single place where all the examples are
+# listed. We want to run these examples in the continuous integration builds
+# because they rot otherwise.
+function run_all_table_admin_examples() {
+  if [ ! -x ../examples/bigtable_samples ]; then
+    echo "Will not run the examples as the examples were not built"
+    return
+  fi
+
+  local project_id=$1
+  shift
+  local zone_id=$1
+  shift
+
+  local ignore_unimplemented="/bin/false"
+  if [ ! -z "${BIGTABLE_EMULATOR_HOST:-}" ]; then
+    ignore_unimplemented="echo Ignoring error as the emulator does not implement this function"
+  fi
+
+  # Create a (very likely unique) instance name.
+  local -r INSTANCE="in-$(date +%s)"
+
+  # Create an instance to run these examples.
+  ../examples/bigtable_samples_instance_admin create-instance "${project_id}" "${INSTANCE}" "${zone_id}" || ${ignore_unimplemented}
+
+  echo
+  echo "Run create-table example."
+  ../examples/bigtable_samples create-table "${project_id}" "${INSTANCE}" sample-table
```

Comment:
If you're going to use the literal `sample-table` everywhere, do you want to put that into a const var?